conf/app.routes

```diff
@@ -533,6 +533,11 @@ POST        /:taxYear/:businessId/capital-allowances/goods-vehicles/check
 
 ########## Capital Allowances - Electric Vehicle Charge Points ##########
 
+GET         /:taxYear/:businessId/self-employment/capital-allowances/electric-vehicle-charge-points/use-allowance                                              controllers.journeys.capitalallowances.electricVehicleChargePoints.ChargePointTaxReliefController.onPageLoad(taxYear: TaxYear, businessId: BusinessId, mode: Mode = NormalMode)
+POST        /:taxYear/:businessId/self-employment/capital-allowances/electric-vehicle-charge-points/use-allowance                                              controllers.journeys.capitalallowances.electricVehicleChargePoints.ChargePointTaxReliefController.onSubmit(taxYear: TaxYear, businessId: BusinessId, mode: Mode = NormalMode)
+GET         /:taxYear/:businessId/self-employment/change-capital-allowances/electric-vehicle-charge-points/use-allowance                                      controllers.journeys.capitalallowances.electricVehicleChargePoints.ChargePointTaxReliefController.onPageLoad(taxYear: TaxYear, businessId: BusinessId, mode: Mode = CheckMode)
+POST        /:taxYear/:businessId/self-employment/change-capital-allowances/electric-vehicle-charge-points/use-allowance                                        controllers.journeys.capitalallowances.electricVehicleChargePoints.ChargePointTaxReliefController.onSubmit(taxYear: TaxYear, businessId: BusinessId, mode: Mode = CheckMode)
```

Comment:
This should be `electric-vehicle-charge-points/buy`, 'use-allowance' is for 7401